react/features/recording/components/Recording/AbstractRecordButton.js

```diff
@@ -0,0 +1,137 @@
+// @flow
+
+import {
+    createToolbarEvent,
+    sendAnalytics
+} from '../../../analytics';
+
+import { openDialog } from '../../../base/dialog';
+import { JitsiRecordingConstants } from '../../../base/lib-jitsi-meet';
+import {
+    getLocalParticipant,
+    isLocalParticipantModerator
+} from '../../../base/participants';
+import {
+    AbstractButton,
+    type AbstractButtonProps
+} from '../../../base/toolbox';
+
+import { getActiveSession } from '../../functions';
+
+import StartRecordingDialog from './StartRecordingDialog';
+import StopRecordingDialog from './StopRecordingDialog';
+
+/**
+ * The type of the React {@code Component} props of
+ * {@link AbstractRecordButton}.
+ */
+export type Props = AbstractButtonProps & {
+
+    /**
+     * True if there is a running active recording, false otherwise.
+     */
+    _isRecordingRunning: boolean,
+
+    /**
+     * The redux {@code dispatch} function.
+     */
+    dispatch: Function,
+
+    /**
+     * The i18n translate function.
+     */
+    t: Function
+};
+
+/**
+ * An abstract implementation of a button for starting and stopping recording.
+ */
+export default class AbstractRecordButton<P: Props>
+    extends AbstractButton<P, *> {
+    accessibilityLabel = 'toolbar.recording';
```

Comment:
Please use toolbar.accessibilityLabel.recording. We have an entire section for accessibility labels.